.travis.yml

```diff
@@ -46,10 +47,12 @@ before_script:
   - gunzip < tests/data/hotosm_bdi_waterways.sql.gz | psql -U postgres test
 
 script:
+  - make -C ./docs html
```

Comment:
Can we build the docs after pytest and before the flake8'ing?

Reply:
Suggestion: more "bash-like" would be: `PYGEOAPI_CONFIG=$(pwd)/pygeoapi-config.yml`

Reply:
Changes done